.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: check-ast
```

Comment:
There is no ast
```suggestion
```

Reply:
This is why I get for copying the sunpy one.